.github/workflows/upload-rpm-images.yml

```diff
@@ -0,0 +1,56 @@
+name: Upload package creation Docker images - RPM - x86 and i386
```

Comment:
Review the `upload-deb-images`  comments if it applies here.

Reply:
Everything solution applies to both files